docs/efun/async/async_db_exec.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: async / async_db_exec
+---
+
+### NAME
+
+    async_db_exec() - executes an sql statement then executes a callback
+
+### SYNOPSIS
+
+    void async_db_exec( int handle, string sql_query, function callback );
+
+### DESCRIPTION
+
+    This function will execute the passed sql statement for the given data‐
+    base handle.
+
+    Returns the number of rows in result set on success, or an error string
+    otherwise, to the callback function provided.
+
+    The callback should follow this format:
+
+        function (int conn) {
```

Comment:
Should this not also take a mixed argument? int | string?

Reply:
I'm not familiar enough with src/packages/async/async.cc to find where it is evaluating the callback with arguments. The existing testsuite code only has an integer argument in its callback for the database connection number.

Reply:
It's because you have written:

```
    Returns the number of rows in result set on success, or an error string
    otherwise, to the callback function provided.
```
which says it's an int or a string (int for the connection handle, string for the error)
but in the example provided it only has the connection handle. i'm not suggesting that it be multiple parameters, only that the parameter should be mixed, since it could be int or string.

Reply:
Ah good catch, that was a copy/paste error from the `db_exec.md` doc file which for some reason I edited without realizing it wasn't relevant for this efun.